QUEST_LV_0100.tsv

```diff
@@ -4679,41 +4679,41 @@ QUEST_LV_0100_20150317_004678	Learn how to earn skills from the Search Scout.
 QUEST_LV_0100_20150317_004679	Learn new skills.
 QUEST_LV_0100_20150317_004680	key to open the Skill window and learn skills. Try learning a skill.
 QUEST_LV_0100_20150317_004681	You have successfully learned a new skill. Talk to the Soldier again
-QUEST_LV_0100_20150317_004682	Talk to the stone statue of the protector
+QUEST_LV_0100_20150317_004682	$Talk to the Protector Statue
 QUEST_LV_0100_20150317_004683	Defeat the polluted Guardian
 QUEST_LV_0100_20150317_004684	A few Guardians have been already polluted due to the vicious energy that pervaded due to the flow of the magic. Destroy the polluted Guardians.
 QUEST_LV_0100_20150317_004685	Contaminated guardian I was completely exterminate. Talk to the guardian of the stone statue.
 QUEST_LV_0100_20150317_004686	Defeat the polluted Vikaras Mage
-QUEST_LV_0100_20150317_004687	Collect the condensed magic source
-QUEST_LV_0100_20150317_004688	The stone statue of the protector told you that you should defeat them using the condensed magic source before the polluted protectors that are sleeping wake up. Collect the condensed magic source. 
-QUEST_LV_0100_20150317_004689	You collected enough of it. Go back to the stone statue of the protector.
-QUEST_LV_0100_20150317_004690	You have collected the condensed magic source to destroy the polluted protectors that are in sleep. Talk to the stone statue of the protector.
+QUEST_LV_0100_20150317_004687	$Collect condensed magic
+QUEST_LV_0100_20150317_004688	$The Protector Statue told you that you should defeat sleeping protectors using condensed magic before they wake up. Collect the condensed magic.
+QUEST_LV_0100_20150317_004689	$You collected enough. Return to the Protector Statue.
+QUEST_LV_0100_20150317_004690	$You have collected the condensed magic used to destroy the corrupted protectors that are in sleep. Talk to the Protector Statue.
 QUEST_LV_0100_20150317_004691	{memo X}Use the condensed magic source to the protectors that are in sleep
-QUEST_LV_0100_20150317_004692	Use the condensed magic before the protectors wake up and spread the pollution.
-QUEST_LV_0100_20150317_004693	You destroyed all the protectors that are in sleep. Go back to the stone statue of the protector.
+QUEST_LV_0100_20150317_004692	$Use the condensed magic on the sleeping protectors to kill them and prevent the spread of corruption.
```

Comment:
sleeping protectors -> sleeping guardians
to kill them? I think it's "before they wake up/awaken"